src/main/kotlin/cacophony/semantic/syntaxtree/ASTGeneration.kt

```diff
@@ -142,6 +142,32 @@ private fun operatorRegexToAST(children: List<ParseTree<CacophonyGrammarSymbol>>
     }
 }
 
+private fun callAndFieldRegexToAst(children: List<ParseTree<CacophonyGrammarSymbol>>, diagnostics: Diagnostics): Expression {
+    val childNum = children.size
+    if (childNum == 1) {
+        return generateASTInternal(children[0], diagnostics)
+    } else {
```

Comment:
**nitpick:** this `else` is redundant since the `if` has a `return` in it and only makes the code more indented